src/OpenTelemetry.Exporter.InMemory/InMemoryExporter.cs

```diff
@@ -37,7 +39,26 @@ internal InMemoryExporter(ExportFunc exportFunc)
 
         internal delegate ExportResult ExportFunc(in Batch<T> batch);
 
-        public override ExportResult Export(in Batch<T> batch) => this.onExport(batch);
+        public override ExportResult Export(in Batch<T> batch)
+        {
+            if (this.disposed)
+            {
+                throw new ObjectDisposedException(this.GetType().Name);
```

Comment:
nit: Maybe add a comment/note here saying this is generally not a good practice for an OTel SDK but the working assumption is in-memory exporter is for testing only so we allow the break with convention here?

Reply:
Good point, update, please help to check my English.